src/zoktai/any.md

```diff
@@ -87,13 +99,15 @@ Backtrack and take the stairs, kill all Ghouls as you proceed to reach **Lvl 4**
 In the 5th room, knock on the wall near the center of the room, then hug the right wall to avoid the Zombie.  
 Use ![][icon_flame] Flame to light the torches.
 
-### <span class="puzzle">Lamp Puzzle</span>
+### <span class="puzzle">Torch Puzzles</span>
+
+Push the Blocks as shown here, then touch the lamp:  
+![](./assets/images/basement_puzzle.webp)
 
-![](./assets/images/basement_puzzle.webp)  
-Use ![][icon_flame] Flame to light the torches, and hammer on levers to proceed.  
-After the dialogs, put all stat points into STRG.
+In the next rooms, use ![][icon_flame] Flame with ![][icon_gladius] Sword to light torches and ![][icon_club] Club on levers to proceed.  
+After the Otenko dialog, put all stat points into STRG.
 
-### <span class="trap">Trap:</span> Skeletons
+### <span class="trap">Skeletons Trap</span>
 
 - Use ![][icon_sol] Sol and hit the Skeletons from behind.
 - Stand on the third Skeleton's position as it dies in case you get a random drop.
```

Comment:
Remove this line for Short Sword RNG Manip route. With no random weapon drops, the Short Sword goes into inventory slot 3, which is the optimal spot for Katana Manip later.